amundsen_application/static/js/components/common/Badges/styles.scss

```diff
@@ -1,10 +1,25 @@
 // Copyright Contributors to the Amundsen project.
 // SPDX-License-Identifier: Apache-2.0
-
 @import 'variables';
-
 .clickable-badge {
   &:hover {
     cursor: pointer;
   }
-}
+  .label {
+    overflow: hidden;
+    padding: 0 0;
+  }
+  .badge-overlay-primary {
+    font-size: 14px;
```

Comment:
Could you use $font-size-base instead?

Reply:
Sounds good, will make sure this is consistent in Flag as well